superset/databases/utils.py

```diff
@@ -113,6 +113,6 @@ def make_url_safe(raw_url: str) -> URL:
     :return:
     """
     try:
-        return make_url(raw_url.strip())
+        return make_url(raw_url.strip())  # noqa
```

Comment:
This seems like a pretty easy way to get around this filter :)

Reply:
Shhh, it's secret!